src/status_im/contexts/wallet/share_community_channel/view.cljs

```diff
@@ -0,0 +1,52 @@
+(ns status-im.contexts.wallet.share-community-channel.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im.contexts.wallet.share-community-channel.style :as style]
+    [utils.i18n :as i18n]
+    [utils.image-server :as image-server]
+    [utils.re-frame :as rf]))
+
+(def qr-size 500)
+
+(defn view
+  []
+  (let [padding-top (:top (safe-area/get-insets))]
+    (fn []
+      (let [chat                (rf/sub [:chats/current-chat-chat-view])
+            {:keys [url]}       (rf/sub [:get-screen-params])
+            qr-media-server-uri (image-server/get-qr-image-uri-for-any-url
+                                 {:url         url
+                                  :port        (rf/sub [:mediaserver/port])
```

Comment:
Better to move the sub to the let declarations, cause we might switch to hooks eventually.